public/js/query.js

```diff
@@ -45,16 +45,18 @@ export class ReportQuery extends Component {
         return this.props.query.hits.length;
     }
     headerJSX() {
-        var meta = `length: ${this.queryLength().toLocaleString()}`;
+        var length = `length: ${this.queryLength().toLocaleString()}`;
+        var meta = length.charAt(0).toUpperCase() + length.slice(1);
```

Comment:
Yes, that's right, just make it simpler with `length` -> `Length`. I'll update it :rocket: 

I thought before, we need dynamic code, without live updates to `Length` or anything, so I made it like now, so that later whatever the sentence is will always be capitalized.